src/launchpad/api/update_api_models.py

```diff
@@ -15,11 +16,22 @@ class AppleAppInfo(BaseModel):
     # TODO: add "date_built" field once exposed in 'AppleAppInfo'
 
 
+class ExtraInfo(BaseModel):
```

Comment:
I don't think this should be its own model, these fields being stored in the `extras` dict is an implementation detail on the model and not something we need to worry about here. This data can be inlined to `UpdateData` IMO.